precompile/precompileconfig/config.go

```diff
@@ -25,3 +31,60 @@ type Config interface {
 	// Verify is called on startup and an error is treated as fatal. Configure can assume the Config has passed verification.
 	Verify() error
 }
+
+// PredicateContext is the context passed in to the PrecompilePredicater interface.
```

Comment:
comment should be updated to match struct name (or we can keep the shorter names)